### ISSUE TITLE
Distributed tracing data ingest discrepancy

### ISSUE THREAD

**Issue by kaylareopelle** [newrelic/newrelic-ruby-agent#871]
Version 8.0 of the agent turns on distributed tracing by default. The distributed tracing team and internal overhead testing indicated this would have a nominal impact on the agent's performance and overall data ingest. However, some customers have noticed a 4x change in the amount of data ingested after upgrading their agent.

### Central Question
* Why did this customer have such a large increase in ingested data with distributed tracing turned on?
* How many other customers might be in the same boat? 

### Resources
_Internal links_
* [#ruby-agent thread from 2021-11-19](https://newrelic.slack.com/archives/C050GQJM4/p1637362666072800)
* [Customer Success ticket: RUBY-2462](https://newrelic.atlassian.net/browse/RUBY-2462)
* [#tmp-agents-dt-by-default discussion about measuring ingest change with DT on by default](https://newrelic.slack.com/archives/C0290QGR02V/p1638476294006300)
* [Trend of unique account IDs with DT](https://onenr.io/08dQeX08Yje)
* [Total # of spans sent to NR in the last 30 days](https://onenr.io/0DvwB4M3Pjp)

**Comment by angelatan2** [comment]
12/10 - Working through a rails test app to finish up our assessment of data increase.

**Comment by kreopelle** [comment]
This ticket has been resolved internally. In our research, we found the specific customer who raised the concern has unique attributes in their application's setup that caused the data ingest size increase. Our testing confirmed maxing out the span reservoir at the default limit of 2000 spans did not cause a large data ingest change for Rails applications. We maintain our original announcement that turning DT on by default should have a minimal impact on most customers.

If any customer has experienced a large jump in their data ingest by turning DT on by default, we encourage them to bring this to our attention to investigate potential bugs.
